Anuja_Pardeshi/HTML/ASSIN3/Contact-Us.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    body{
+        background-color: darkgray;
+    }
+</style>
+<body>
+    <div style="text-align: center;">
+        <h1>AmberSoft</h1><br>
+        <a href="#" >Home </a> |
```

Comment:
There should be path for homepage  like `<a href="Que2.html>Home</a>`

Reply:
Is it ok now ? I have Added the Path

Reply:
yes all good.